.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
I would make this ignore more focused:

    /node_modules/

Reply:
I've made this change in 66f37948e5c72ab3b3deb9998c8482b2c94b5bf6. Good suggestion!